dev-client/src/screens/SoilScreen/components/SoilSurfaceScreen.tsx

```diff
@@ -59,16 +65,12 @@ export const SoilSurfaceScreen = ({siteId}: Props) => {
         </Heading>
         <LastModified />
         <Select
-          selectedValue={cracking ?? undefined}
-          onValueChange={onUpdate as (_: string) => void}>
-          {surfaceCracks.map(crack => (
-            <Select.Item
-              key={crack}
-              value={crack}
-              label={t(`soil.vertical_cracking.value.${crack}`)}
-            />
-          ))}
-        </Select>
+          nullable
+          value={cracking ?? null}
```

Comment:
question: do you need the `?? null`? Do we need to have `null` instead of `undefined`?

Reply:
well we designed the API for `Select` so we don't _need_ to have it, but `null` felt more semantically appropriate to me and allowing either felt a bit disorganized